test/Common/TestActivities.cs

```diff
@@ -53,7 +53,7 @@ public static long GetFileSize([ActivityTrigger] DurableActivityContext ctx)
         public static void ThrowActivity([ActivityTrigger] DurableActivityContext ctx)
         {
             string message = ctx.GetInput<string>();
-            throw new Exception(message);
+            throw new InvalidOperationException(message);
```

Comment:
Was this change required? I thought `SubOrchestrationThrow` calls `ThrowOrchestrator`, not `ThrowActivity`.

Reply:
This is not required. But I added it because at one point we were having issues where exceptions were not being deserialized to the correct type. Throwing a non-generic Exception would have helped us catch that, so I figure it's worth adding it now.